src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.data;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
+	
+	private Person person;
+	private ArrayList<String> tagHistory;
+	
```

Comment:
Tagging is for the relation between Person and Tag, so Person and Tag should be stored instead, as discussed in class